susemanager-utils/testing/automation/wait-for-builds.sh

```diff
@@ -9,6 +9,7 @@ usage() {
 api=https://api.opensuse.org
 config_file=$HOME/.oscrc
 lock="yes"
+TIMEOUT="2h"
```

Comment:
I'm fine with it, but could we use downcase as the others? Or uppercase in all of them :P